cli.js

```diff
@@ -35,6 +37,7 @@ const cli = meow(`
     --json         Output results as JSON
     --mobile       Emulates a Nexus 5X (default=true)
     --load-page    Loads the page (default=true)
+    --output       Outputs an HTML report to the specified path
```

Comment:
Done. Made output the flag, with options on html, json, and default is prettyPrint.